stdlib/LinearAlgebra/src/hessenberg.jl

```diff
@@ -90,6 +90,10 @@ Base.isassigned(H::UpperHessenberg, i::Int, j::Int) =
 Base.@propagate_inbounds getindex(H::UpperHessenberg{T}, i::Integer, j::Integer) where {T} =
     i <= j+1 ? convert(T, H.data[i,j]) : zero(T)
 
+Base._reverse(A::UpperHessenberg, dims) = reverse(Matrix(A); dims)
+Base._reverse(A::UpperHessenberg, dims::Colon) = (reverse(Matrix(A)), A.data)
```

Comment:
Should this just be this?
```suggestion
Base._reverse(A::UpperHessenberg, dims::Colon) = reverse(Matrix(A))
```

I don't know why we would return a tuple, maybe I'm missing something.